web/src/features/map/MapTooltip.tsx

```diff
@@ -4,9 +4,10 @@ import { useAtom } from 'jotai';
 import useGetState from 'api/getState';
 import CarbonIntensitySquare from 'components/CarbonIntensitySquare';
 import { CircularGauge } from 'components/CircularGauge';
-import { getSafeTooltipPosition } from 'components/tooltips/utilities';
 import { ZoneName } from 'components/ZoneName';
+import { getSafeTooltipPosition } from 'components/tooltips/utilities';
 import { useTranslation } from 'translation/translation';
+import { StateZoneData } from 'types';
```

Comment:
I'm not sure how this affect things or if it do any difference at all in build sizes but we should consider using import type instead or just plain imports.
```suggestion
import type { StateZoneData } from 'types';
```

Reply:
I think Rollup is smart enough to handle this, especially in this case where the file only exists of types. But not 100% sure. 

But yes, it would generally be a nice improvement in future to clean this up, set up an ESLint rule and get VSCode to import it correctly too :)

Will not delve into this now (as I think we need to do it properly), but thanks for raising it! 🙌

Reply:
Yeah I didn't intend for it to be done in this PR 😅

It was more to highlight it for the future.